presto-main/src/main/java/com/facebook/presto/sql/planner/PlanPrinter.java

```diff
@@ -544,6 +562,25 @@ private void printDistributions(int indent, PlanNodeStats nodeStats)
         return ImmutableMap.of();
     }
 
+    private void printWindowNodeStats(int indent, WindowNodeStats stats)
+    {
+        output.append(indentString(indent));
+        output.append(format("Active Drivers: [ %d / %d ]", stats.getActiveDrivers(), stats.getTotalDrivers()));
+        output.append('\n');
+
+        output.append(indentString(indent));
+        output.append(formatDistribution("Index size in bytes", stats.getIndexSizeDistribution()));
```

Comment:
I don't think displaying detailed distributions is a good idea. First of all, they are not very readable. Additionally, they make already complex `EXPLAIN ANALYZE` output even more detailed and confusing.

I think the best is to show one/two clean metrics which a user could use to quickly check if there is any problem with the data (e.g: skewness). If there is, then he/she could dig deeper using query JSON for instance or diagnose problem by dissecting the query.

I am thinking about:
```
Index size avg: XXX, Index size std.dev: XXX
```
Additional benefit of that is that you can compute those values online and don't have to send all index data to the master.

See for instance:
```
             - InnerJoin[("orderkey" = "orderkey_0")][$hashvalue, $hashvalue_7] => [orderkey:bigint, $hashvalue:bigint, orderkey_0:bigint, $has
                     Cost: 39.15%, Output: 600572 rows (20.62MB)                                                                               
                     Left (probe) Input avg.: 150143.00 lines, Input std.dev.: 0.33%                                                           
                     Right (build) Input avg.: 9375.00 lines, Input std.dev.: 0.80%                                                            
                             Collisions avg.: 6244.50 (99.59% est.), Collisions std.dev.: 5.03%
```
I think it's much clearer than displaying explicit distributions of inputs and collisions. You can also check the code for collisions and input stats to see how stat are computed online